src/components/User/Bookings.jsx

```diff
@@ -52,7 +64,10 @@ export default function Bookings() {
                 { bookings.map(booking => {                
                     return (
                         <div key={booking.id}>
-                            <BookingCard booking={booking} onDelete={() => deleteBooking(booking.id)} />
+                            <BookingCard
+                                booking={booking}
+                                onDelete={() => deleteBooking(booking.id)}
+                                onConfirm={() => confirmBooking(booking.id)} />
```

Comment:
I recommend passing the entire function, like this 
`onDelete={deleteBooking}` 
`onConfirm={confirmBooking}`
And on the BookongCard, you can handle passing the booking.id when the buttom is clicked: 
`onClick={() => onConfirm(booking.id)}` 
`onClick={() => onDelete(booking.id)}`
This change will improve performance as the app gets bigger, 
it will prevent the parent component (Bookings.jsx)from creating new functions every time it re-renders.
When new functions are created unnecessarily, it can trigger updates to the `BookingCard` component
even if nothing has actually changed.
Passing the whole function and handling the `id`in the `onClick` within `BookingCard, we avoid this issue.